internal/renderers/femtovg/itemrenderer.rs

```diff
@@ -642,7 +652,12 @@ impl<'a> ItemRenderer for GLItemRenderer<'a> {
     ///  * Blur the image
     ///  * Fill the image with the shadow color and SourceIn as composition mode
     ///  * Draw the shadow image
-    fn draw_box_shadow(&mut self, box_shadow: Pin<&items::BoxShadow>, item_rc: &ItemRc) {
+    fn draw_box_shadow(
+        &mut self,
+        box_shadow: Pin<&items::BoxShadow>,
+        item_rc: &ItemRc,
+        _size: LogicalSize,
```

Comment:
Unfortunately, the box shadow is cached and we need to re-query the with and height within the cached function to ensure it is re-drawn.  (Some other item have the same problem like the native controls, and the layer).
This won't help to take the size away from the item :-/

Reply:
Yeah, we'll still need that ability, but we can still expose that in ItemRc for example, no? (Then call the vtable function)